ui/js/models/resultsets_store.js

```diff
@@ -204,6 +204,9 @@ treeherder.factory('ThResultSetStore', [
 
                 name:repoName,
 
+                // keep track of the selected job here, because the DOM may
+                // lose track when we update with new jobs and have to re-render
+                // a platform row with new jobs that have come in.
```

Comment:
I wonder if we can wordsmith this a bit, I had a bit of a hard time grok'ing the last half.

Maybe something like:
*"This keeps track of the selected job. There are some cases where the DOM may fail to do so during job ingestion, polling, and rendering in the job row."*

Or whatever makes sense :)